[terraform] add azure_management_group_id var to azure discovery module (#67215)
gravitational/teleport: docs/pages/reference/infrastructure-as-code/terraform-modules/teleport-discovery-azure/examples/management-group.mdx

---

> title: Example for discovering Azure resources in a single account

Reviewer: ```suggestion
title: Example for discovering Azure resources
```

Reviewer 2: or maybe "in a management group"

sidebar_label: management-group

> description: Configure Teleport to discover resources in a Azure account.

Reviewer 2: ```suggestion
description: Configure Teleport to discover resources in an Azure management group.
```
